pyglider/seaexplorer.py

```diff
@@ -485,18 +488,26 @@ def raw_to_timeseries(indir, outdir, deploymentyaml, kind='raw',
     except:
         pass
     id0 = ds.attrs['deployment_name']
-    outname = outdir + id0 + fnamesuffix + '.nc'
-    _log.info('writing %s', outname)
     if 'units' in ds.time.attrs.keys():
         ds.time.attrs.pop('units')
     if 'calendar' in ds.time.attrs.keys():
         ds.time.attrs.pop('calendar')
     if 'ad2cp_time' in list(ds):
         if 'units' in ds.ad2cp_time.attrs.keys():
             ds.ad2cp_time.attrs.pop('units')
-    ds.to_netcdf(outname, 'w',
-                 encoding={'time': {'units': 'seconds since 1970-01-01T00:00:00Z',
-                                    'dtype': 'float64'}})
+    if og_format:
+        ds = utils.add_og1_metadata(ds, deployment)
+        outname = f"{outdir}/{ds.attrs['id']}.nc"
+        _log.info('writing %s', outname)
+        ds.to_netcdf(outname, 'w',
+                     encoding={'TIME': {'units': 'seconds since 1970-01-01T00:00:00Z',
+                                        'dtype': 'float64'}})
+    else:
+        outname = outdir + id0 + fnamesuffix + '.nc'
+        _log.info('writing %s', outname)
+        ds.to_netcdf(outname, 'w',
+                     encoding={'time': {'units': 'seconds since 1970-01-01T00:00:00Z',
```

Comment:
For example, 'time'/'TIME' could be replaced by a time-encoder yaml entry. eg calls to `ds.time` could be replaced by `ds[time_var_name]` or similar?  

Not sure about 'latitude' etc, but why do they need to be translated back and forth?  For distance over ground?  Maybe just use inside that function?

Reply:
I can localise lon/lat to just change within those functions, I'll make a new PR with `ds[time_var_name]` and see what it looks like